.travis.yml

```diff
@@ -12,5 +12,5 @@ cache:
 install: true
 
 script:
-  - ./mvn-build-all.sh clean install jacoco:report
-  - mvn -U coveralls:report
+  - mvn -U clean install jacoco:report
```

Comment:
- [ ] Do you need to apply the jacoco to all of the modules yet?

For example, I would configure as follows.
* terasoluna-gfw-parent/pom.xml

    remove configuration of `maven-surefire-plugin` and `jacoco-maven-plugin`
* terasoluna-gfw-common-libraries/pom.xml

    add configuration of `maven-surefire-plugin` and `jacoco-maven-plugin`
* root pom.xml

    add configuration of `maven-surefire-plugin` and `jacoco-maven-plugin` in `common-libs-only` profile
* travis.yml
```
mvn -U clean install
mvn -U -P common-libs-only jacoco:report coveralls:report
```